ui/src/app/subnets/views/SubnetDetails/SubnetSummary/SubnetSummaryForm/SubnetSummaryFormFields/SubnetSummaryFormFields.tsx

```diff
@@ -73,23 +46,12 @@ const SubnetSummaryFormFields = ({
           name="allow_dns"
           type="checkbox"
         />
-        {values.fabric && (
-          <FabricSelect
-            name="Fabric"
-            defaultOption={{
-              label: getFabricDisplay(fabric) || "",
-              value: values?.fabric.toString() || "",
-            }}
-          />
-        )}
+        <FabricSelect name="fabric" />
```

Comment:
The main problem here initially was that the `name` prop was "Fabric" instead of "fabric". The `name` prop has to correspond to a form value name exactly. So this is one reason why the fabric never updated the VLAN options.

We could probably handle this a bit better with TS generics so that it's impossible to use an invalid `name`, but for now we just have to remember to get it right.